### PR TITLE
community/php5: remaded to the same as php7

### PR DESCRIPTION
**Pull request by vakartel**
Remaded APKBUILD and related files to the same as in current community/php7

### CODE REVIEW

**Comment by vakartel** [comment]
@jirutka 
I remaded it with php7/APKBUILD you made and add all php5-specific code and patches.
Somehow mostly all tests are failed so I just disable check phase.
So if it pass, php7 and php5 would have identical behavior and any future changes in APKBUILD need to be done in both php5 and php7 ones.

**Comment by skwashd** [comment]
@jirutka / @vakartel any news on this one? I would like to get #1572 merged so composer can be used with either PHP 5.6 or 7.x. Should this continue to be a blocker for for the meta package PR?

**Comment by andypost** [comment]
@vakartel that's still makes sense

**Comment by skwashd** [comment]
@andypost what is the plan for supporting PHP 5.6? Security support ends in less than 10 months.

**Comment by jirutka** [comment]
> @vakartel that's still makes sense

IMHO it’s not worth changing now. As @skwashd wrote, 5.6 is near EOL, so we will drop it after after v3.8 release.

File: community/php5/APKBUILD [hunk 1]
```diff
@@ -1,514 +1,652 @@
...
-		--without-db1 \
-		--without-db2 \
-		--without-db3 \
-		--without-qdbm \
-		--with-mssql=shared \
-		--with-pdo-dblib=shared \
-		--enable-opcache \
-		"
+			--with-libzip=/usr \
+		--with-zlib=shared \
```
Comment (andypost) [inline community/php5/APKBUILD]: zlib should be bundled https://github.com/alpinelinux/aports/pull/2951

**Comment by vakartel** [comment]
IMHO this PR have no sense now because EOL

**Comment by andypost** [comment]
EOL is a year ahead so before 3.8 better to have php5 maintainable

**Comment by jirutka** [comment]
Applying single commit with +955/-407 changes, which is basically impossible to review, would not make it better maintainable. It’s IMHO not worth the risk of possible breakages and unclear changes in git log.

**Pull request closed without merging**